### PR TITLE
[SDEV-1938] make v em gui compatible with ubuntu 22

### PR DESCRIPTION
**Pull request by nandishjpatel**
Fixed:

In xrc file `<bg>#4D4D4D</bg>` (The bg of CaptionBar and acq button was missing)
Screenshot from 2024-12-06 08-24-31

In xrc file `<border>` (No calibration run text was cropped)
Screenshot from 2024-12-06 08-24-06

Settings dwell time (The slider was cropped)
Screenshot from 2024-12-06 08-26-36

Immersion mode cbox (the cbox position not aligned properly)
Screenshot from 2024-12-06 08-23-46

### CODE REVIEW

File: src/odemis/gui/cont/stream.py [hunk 1]
```diff
@@ -1386,12 +1388,33 @@ def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=Non
         assert isinstance(stream, acqstream.SEMStream)  # don't show for CCD stream
         self._add_fastem_ctrls()
 
+    def _create_immersion_mode_cbox(self):
+        # HACK manually create the checkbox using custom gb_sizer's pos and span
```
Comment (tepals) [inline src/odemis/gui/cont/stream.py:1391]: can you add to this comment why this hack is necessary?

File: src/odemis/gui/cont/stream.py [hunk 2]
```diff
@@ -1392,10 +1391,11 @@ def _create_immersion_mode_cbox(self):
         # HACK manually create the checkbox using custom gb_sizer's pos and span
         _ = self.stream_panel._add_side_label("Immersion Mode")
         cbox_immersion_mode = wx.CheckBox(self.stream_panel._panel, wx.ID_ANY,
-                                 style=wx.ALIGN_RIGHT | wx.NO_BORDER)
+                                          style=wx.ALIGN_RIGHT | wx.NO_BORDER)
 
         self.stream_panel.gb_sizer.Add(cbox_immersion_mode, (self.stream_panel.num_rows, 2), span=(1, 1),
-                          flag=wx.ALIGN_CENTRE_VERTICAL | wx.EXPAND | wx.TOP | wx.BOTTOM, border=5)
+                                       flag=wx.ALIGN_CENTRE_VERTICAL | wx.EXPAND | wx.TOP | wx.BOTTOM,
+                                       border=5)
```
Comment (tepals) [inline src/odemis/gui/cont/stream.py:1398]: These changes should be in the other commit if you want to keep two separate commits

**Review by tepals — CHANGES_REQUESTED** [review]
only some small comments, I checked it on Ubuntu 20.04 and it still looks good there as well

File: src/odemis/gui/cont/stream.py (hunk 1 above)
Comment (nandishjpatel) [reply src/odemis/gui/cont/stream.py:1391]: Comment added

**Review by tepals — APPROVED** [review]

File: src/odemis/gui/xmlh/resources/panel_tab_fastem_acqui.xrc [hunk 3]
```diff
@@ -222,6 +222,7 @@
                     <border>10</border>
                   </object>
                 </object>
+                <bg>#4D4D4D</bg>
```
Comment (tmoerkerken) [inline src/odemis/gui/xmlh/resources/panel_tab_fastem_acqui.xrc:225]: I'm kind of surprised we hardcode styling like this. Commonly in UI programming sizes and colors are defined once, and then reused via variables. This greatly enhances consistency. There might be a variable system in xrc that could facilitate this. Obviously, not in scope for your work now.

Comment (nandishjpatel) [reply src/odemis/gui/xmlh/resources/panel_tab_fastem_acqui.xrc:225]: You are right! I quickly checked and it seems can one cannot define a constant in a XRC file. Maybe we can set an object's attribute e.g.
```
<attribute name="obj_bg">CONSTANT</attribute>
```

This is some refactoring work maybe for later. Can you approve this PR for now, if convinced @tmoerkerken ?

Comment (tmoerkerken) [reply src/odemis/gui/xmlh/resources/panel_tab_fastem_acqui.xrc:225]: Agreed it is completely out of scope for your work. Will approve. Might be an interesting topic for a Knowledge Sharing topic though 🤔

**Review by tmoerkerken — APPROVED** [review]
